bofire/data_models/outlier_detection/outlier_detections.py

```diff
@@ -0,0 +1,51 @@
+import itertools
+from typing import List
+
+from bofire.data_models.base import BaseModel
+from bofire.data_models.domain.api import Inputs, Outputs
+from bofire.data_models.outlier_detection.outlier_detection import IterativeTrimming
+
+AnyOutlierDetector = IterativeTrimming
+
+
+class OutlierDetections(BaseModel):
+    """ "List of Outlier detectors.
+
+    Behaves similar to a outlier_detector."""
+
+    detectors: List[AnyOutlierDetector]
+
+    @property
+    def outputs(self) -> Outputs:
+        return Outputs(
+            features=list(
+                itertools.chain.from_iterable(
+                    [model.outputs.get() for model in self.detectors]  # type: ignore
+                )
+            )
+        )
+
+    def _check_compability(self, inputs: Inputs, outputs: Outputs):
```

Comment:
Can you add tests for the errors raised here? You can get inspiration from the tests from `BotorchSurrogates` when we test the compatibility method there.